tsdb/mapper.go

```diff
@@ -39,7 +39,8 @@ func (mo *mapperOutput) key() string {
 // LocalMapper is for retrieving data for a query, from a given shard.
 type LocalMapper struct {
 	shard           *Shard
-	stmt            *influxql.SelectStatement
+	stmt            influxql.Statement
```

Comment:
One way the previous code got difficult to follow was the number of attributes on the Mapper type. I'd like to suggest something like this going forward:

```
type MapperStatement struct {
    stmt influxql.Statement
    selectStmt *influxql.SelectStatement
    raw bool
}
```
This makes it very clear what attributes work together, and the logic that binds them. Make sense?